lib/cucumber/glue/snippet.rb

```diff
@@ -81,6 +81,8 @@ def self.example(cucumber_expression_generator)
           new(cucumber_expression_generator, 'Given', 'I have 2 cukes', MultilineArgument::None.new).step
         end
 
+        private_class_method :example
```

Comment:
Actually made the example method private with this line. Test still pass so ideally we're all good.

Reply:
The only usage _here_ seems to be in the CLI options-creating one-liner.

I'm not fully familiar with the `glue/` code - I can not say whether this is used anywhere else. What the code _does_ indicates that it's quite local. 

Alternative: If it truly is local, it could be made a local variable instead of its own method.

Alternative 2: Do not change the visibility of this method in this PR.

Reply:
With the way it was written before, it was written under the `private` section, but self methods aren't private unless specific set private. One would assume that if it was under private that it should be private, yeah? [Info here](http://www.rubydoc.info/github/bbatsov/RuboCop/RuboCop/Cop/Lint/IneffectiveAccessModifier)

Reply:
That's a good indication, supporting "it's local".

Reply:
@olleolleolle Interestingly enough, if I change step 47 to `format('%-7s: %-28s e.g. %s', type, description, "")` and comment the rest out, the tests pass still.

Reply:
This makes me wonder if perhaps there's a coverage issue?

Reply:
@olleolleolle Any direction given my most recent comments above? Perhaps I should just leave this one be for now and create another issue to investigate it further? /shrug

Reply:
Let's leave this out of this change - you're right.

Reply:
Will do!